jenkins/databricks/init_cudf_udf.sh

```diff
@@ -20,8 +20,7 @@
 
 set -ex
 
-# TODO: https://github.com/NVIDIA/spark-rapids/issues/12021
-CUDF_VER=${CUDF_VER:-25.02}
+CUDF_VER=${CUDF_VER:-25.04}
```

Comment:
Draft the PR, because currently `25.04` CUDF nightly python packages are not available.

To merge this PR, we can:

    - Either wait for `25.04` CUDF nightly python packages available here:
       https://anaconda.org/rapidsai-nightly/cudf/files.
 
    - Or revert to `CUDF_VER=25.02`, merge this PR first; 
       then file another PR updating file init_cudf_udf.sh after `25.04` cuDF python packages are available.